contracts/libraries/VaultLifecycleTreasury.sol

```diff
@@ -225,19 +255,18 @@ library VaultLifecycleTreasury {
                 .add(withdrawAmountDiff);
         }
 
-        {
-            (performanceFeeInAsset, , totalVaultFee) = getVaultFees(
+        // We do not charge on the first round
+        managementFeeInAsset = vaultState.round <= 1
```

Comment:
Instead of adding this `vaultState.round <= 1` check that will run on all subsequent rounds, we could instead pass in `params.managementFee = 0`. This will help us avoid looking up `vaultState.round` from storage too